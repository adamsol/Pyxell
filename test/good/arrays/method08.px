
a = [for i in 1..10^^6 yield random.randFloat()]
b = a[::-1]
print a[0] == b[0]

print a.reverse().length
print a == b
