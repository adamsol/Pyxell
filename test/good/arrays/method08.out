false
1000000
true
